distributed-job-manager/src/test/java/ru/fix/distributed/job/manager/WorkPooledMultiJobIT.java

```diff
@@ -63,118 +53,19 @@ public void shouldDistributeCommonJobs() throws Exception {
                 DistributedJobManager jobManager3 = createNewJobManager(nodeIds[2], curator)
         ) {
             String searchedWorkItem = "work-item-1.1";
-            assertTimeout(Duration.ofMillis(30_000),
-                    () -> {
-                        // Work pool contains 3 work items. Then every distributed job should contains 1 work item.
-                        for (String nodeId : nodeIds) {
-                            String assignedWorkpoolPath = paths.assignedWorkPool(nodeId, getJobId(1));
-                            if (curator.checkExists().forPath(assignedWorkpoolPath) != null) {
-                                List<String> workPool = curator.getChildren().forPath(assignedWorkpoolPath);
-                                if (workPool.contains(searchedWorkItem)) {
-                                    return true;
-                                }
-                            }
-                        }
-                        return false;
-                    }, "Wait for assignment work-item-1.1 to any worker");
-        }
-    }
-
-    @Test
-    @Disabled("При повторном создании ProxySocket порт оказывается уже занятым")
```

Comment:
it's hard to maintain that disabled test. I suggest write it from scratch during resolving https://github.com/ru-fix/distributed-job-manager/issues/30